expHome/src/Components/Home/Home.js

```diff
@@ -16,7 +16,7 @@ import sectionsOrder from "./modules/views/sectionsOrder";
 import UniversitiesMap from "./modules/views/UniversitiesMap/UniversitiesMap";
 
 function Index() {
-  const [section, setSection] = useState(-1);
```

Comment:
set null instead of -1

Reply:
> set null instead of -1

what about 0? we need the underline to be under the first tab at the start .